app/uk/gov/hmrc/merchandiseinbaggage/config/AppConfig.scala

```diff
@@ -16,51 +16,44 @@
 
 package uk.gov.hmrc.merchandiseinbaggage.config
 
+import com.google.inject.Inject
+import play.api.Configuration
+import uk.gov.hmrc.merchandiseinbaggage.config.AppConfig._
 import javax.inject.Singleton
-import uk.gov.hmrc.merchandiseinbaggage.config.AppConfigSource._
-import pureconfig.generic.auto._ // Do not remove this
 
 @Singleton
-class AppConfig()
-    extends MongoConfiguration
-    with EmailConfiguration
-    with EoriCheckConfiguration
-    with CurrencyConversionConfiguration {
-  lazy val bfEmail: String = configSource("BF.email").loadOrThrow[String]
-}
+class AppConfig @Inject() (val config: Configuration)() {
```

Comment:
Can this please be cleaned up. Can just Inject bootstrap's `ServicesConfig`. For example, something like the below:

```
@Singleton
class AppConfig @Inject() (servicesConfig: ServicesConfig, config: Configuration) {
  lazy val bfEmail: String               = config.get[String]("BF.email")
  lazy val emailUrl: String              = servicesConfig.baseUrl("email")
  lazy val eoriCheckUrl: String          = servicesConfig.baseUrl("eori-check")
  lazy val currencyConversionUrl: String = servicesConfig.baseUrl("currency-conversion")
}
```

Reply:
Also if you do not want to inject configuration you can update bfEmail to `servicesConfig.getString("BF.email")`